Algorithm.Python/DropboxCoarseFineAlgorithm

```diff
@@ -0,0 +1,70 @@
+class DropboxCoarseFineAlgorithm(QCAlgorithm):
```

Comment:
Please add the open-source headers from the other files:
https://github.com/QuantConnect/Lean/blob/master/Algorithm.Python/AddRemoveSecurityRegressionAlgorithm.py#L1

Please add the file extension -- its missing .py.

This should never have compiled; make sure you build the project before submitting a pull-request